app/views/searches/_form.html.erb

```diff
@@ -2,10 +2,12 @@
   <%= react_component("Filters", @searcher_serializer.as_json.merge({
     controllerName: Settings.filters_beta.present? ? controller_name.to_json : nil,
   })) %>
-  <div class="search-footer">
-    <%= search_examples %>
-    <a href="#"><%= t('search.help_link') %></a>
-  </div>
+  <% unless Settings.filters_beta.present? && controller.is_a?(ResponsesController) %>
```

Comment:
Is it time to get rid of the filters_beta setting check? This thing is ready for full release after this PR, right?